### DISCUSSION TITLE
How to build iPXE with TUI menu?

### DISCUSSION THREAD

**Discussion by gobiursus** [ipxe/ipxe#1218] [General]
Hi,
I am new to iPXE. My card is ConnectX-3-FCBT, how to I build my iPXE with TUI interface? The current firmware I built directly send me to shell.
Thank you in advance!

**Comment by mcb30** [comment]
You can access the configuration user interface via the [`config`](https://ipxe.org/cmd/config) command, and build menus using the [`menu`](https://ipxe.org/cmd/menu) command.

**Reply by gobiursus** [reply]
Thank you!
